Cargo.toml

```diff
@@ -50,11 +51,13 @@ move-vm-test-utils = { git = "https://github.com/aptos-labs/aptos-core", tag = "
 move-vm-types = { git = "https://github.com/aptos-labs/aptos-core", tag = "aptos-node-v1.14.0" }
 moved = { path = "moved" }
 moved-engine-api = { path = "engine-api" }
+moved-rocksdb = { path = "rocksdb" }
 once_cell = "1.19"
 op-alloy = { version = "0.6", features = ["full", "std", "k256", "serde"] }
 openssl = "0.10"
 regex = "1.11.1"
 revm = { version = "17", features = ["optional_balance_check"] }
+rocksdb = { git = "https://github.com/rust-rocksdb/rust-rocksdb", rev = "7f9cba4" }
```

Comment:
Why use a GitHub dependency instead of a regular crates.io one? The crate is [listed here](https://crates.io/crates/rocksdb).

Reply:
This one has a newer RocksDB 9.10, which is not in the published crates.